src/components/RadioButton/RadioButton.visual.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { VisualTestContainer } from '../../../test/visual/VisualTestContainer';
+import { RadioButton } from './';
+
+class RadioButtonVisual extends React.Component<any> {
+  render() {
+    return (
+      <VisualTestContainer>
+        <RadioButton {...this.props} />
+      </VisualTestContainer>
+    );
+  }
+}
+
+const tests = [
+  {
+    describe: 'basic',
+    its: [
+      {
+        it: 'default',
+        props: {},
+      },
+    ],
+  },
+];
+
+tests.forEach(({ describe, its }) => {
+  its.forEach(({ it, props }) => {
+    storiesOf(`RadioButton/${describe}`, module).add(it, () => (
+      <RadioButtonVisual {...props} />
+    ));
+  });
+});
```

Comment:
I know it wasn't part of the generator, but can you please use `storybook-snapper` instead of `storiesOf`.
You can take a look at the `ShareButton` for instance as an example: https://github.com/wix/wix-ui-tpa/blob/ea1685638e03e0d9a401285453c59106b1dfb8c1/src/components/ShareButton/ShareButton.visual.tsx

Also, we should probably have additional test cases, like `disabled` and `checked`

I really need to add this to the generator...
Sorry for the hassle

Reply:
when you have the time to explain all tests it would be great.